File: OmkarFCUBEAPP/src/app/models/tripstatusrptlistmodel.ts
import { Paginationmodel } from "./paginationmodel";
import { Tripstatusrptmodel } from "./tripstatusrptmodel";

export class Tripstatusrptlistmodel {
    tripStatusRptlist: Tripstatusrptmodel[] = [];
    pageMetaData: Paginationmodel = new Paginationmodel;
}

File: OMKARAUTOAPI/FleetTrans/Models/TripStatusRptListModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Shared.Models;

namespace FleetTrans.Models
{
    public class TripStatusRptListModel
    {
        public List<TripStatusRptModel> TripStatusRptlist { get; set; }
        public PaginationMetaData PageMetaData { get; set; }
    }
}
